### ISSUE TITLE
The image-upload can't work in PhoneGap wrapper

### ISSUE THREAD

**Issue by Coralma** [jupenur/lib-gwt-imageupload#1]
Hi,
I try to wrap the website as a phonegap native app. I found the image-upload can work fine in browser but it doesn't work inside the phoneGap app.  Do you have plan to support the phoneGap?

**Comment by jupenur** [comment]
This is actually already being discussed on the Vaadin bugtracker, [here](http://dev.vaadin.com/ticket/13867). The problem is Android's WebView not supporting `<input type="file">`. I'm currently looking into possible workarounds, but it might be that there's nothing that can be done.

**Comment by Coralma** [comment]
Thanks for your reply. Do you know whether it could work fine in IOS device?

**Comment by jupenur** [comment]
I just checked, and yes, on iOS it works perfectly fine.

**Comment by Coralma** [comment]
Do you check within a phoneGap wrapped iOS native app?
I tested It could work in Android chrome and iOS safari broswer. But the phoneGap app of android doesn't work.

**Comment by jupenur** [comment]
Yes. I used the [PhoneGap Developer](https://itunes.apple.com/us/app/phonegap-developer/id843536693) app. The issue only exists in Android's WebView.

**Comment by jupenur** [comment]
FYI: the Vaadin ticket is now closed and I'm moving any discussion here.

**Comment by Coralma** [comment]
OK, I got it. Thank you very much.
